diff -uNr git/common/env_flags.c git2/common/env_flags.c
--- git/common/env_flags.c	2018-05-31 11:01:18.318897927 +0200
+++ git2/common/env_flags.c	2018-05-31 11:02:31.275070110 +0200
@@ -420,7 +420,8 @@
 {
 	const char *var_name = var_entry->key;
 	const char *flags_list = getenv(ENV_FLAGS_VAR);
-	char flags[ENV_FLAGS_ATTR_MAX_LEN + 1] = "";
+	char flags[ENV_FLAGS_ATTR_MAX_LEN + 1];
+	flags[0] = '\0';
 	int ret = 1;
 
 	/* look in the ".flags" and static for a reference to this variable */
