diff --git a/Makefile b/Makefile
index 001375cfd..9e88c5f6a 100644
--- a/Makefile
+++ b/Makefile
@@ -771,6 +771,9 @@ KBUILD_CFLAGS   += $(call cc-option,-Werror=strict-prototypes)
 # Prohibit date/time macros, which would make the build non-deterministic
 KBUILD_CFLAGS   += $(call cc-option,-Werror=date-time)
 
+# Allow format truncation
+KBUILD_CFLAGS   += $(call cc-option,-Wno-error=format-truncation)
+
 # use the deterministic mode of AR if available
 KBUILD_ARFLAGS := $(call ar-option,D)
 
